test/__snapshots__/action.test.ts.snap

```diff
@@ -0,0 +1,144 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`GitHubActionTypeScriptProject source code default source code 1`] = `
+"import * as core from '@actions/core';
+import { loadActionOptions } from './options.generated.ts';
+
+async function run() {
+  const options = loadActionOptions();
+  console.log(options);
+}
```

Comment:
I'd consider adding some comment(s) explaining that loadActionOptions is generated from \<someOption> in the projenrc file, so to change it you should go there, and if you don't want to use it / autogenerate it, you can disable it with \<someOtherOption>.